server/types/domain.go

```diff
@@ -30,16 +30,15 @@ var ErrDomainValueViolatesCheckConstraint = errors.NewKind(`value for domain %s
 // NewDomainType creates new instance of domain DoltgresType.
 func NewDomainType(
 	ctx *sql.Context,
-	schema string,
-	name string,
+	schema, name, owner string,
```

Comment:
Only mentioning it here, but since we're removing ownership, we can remove most references to owners altogether.